New package: wraith-master-1.2.1

adigitoleo opened the pull request:
Package for RGB control of AMD Wraith Master cooling fans. As far as I am aware, the gradle build system is not capable of producing PIE executables, hence the use of `nopie_files`. The GTK3 and cli frontends can be built separately, as per upstream build options (and the AUR PKGBUILD).

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES** (system-wide install, compiled)

#### Local build testing
- I built this PR locally for my native architecture, (glibc x86_64)

adigitoleo commented:
CI can't find the java executable. Build log excerpt:

```
> Task :core:cinteropLibusbLinuxX64 FAILED
226

227
FAILURE: Build failed with an exception.
228

229
* What went wrong:
230
Execution failed for task ':core:cinteropLibusbLinuxX64'.
231
> Process 'command '/usr/lib/jvm/java-1.8-openjdk/bin/java'' finished with non-zero exit value 1
```

After trying a few things like adding openjdk8 to dependencies and setting JAVA_HOME, I'm out of ideas at the moment. It seems to build fine locally but I did run into this error a couple of times before I got it to work. I have no experience with gradle, unfortunately. Maybe @asiekierka can help, who maintains the gradle package.

asiekierka commented:
I'm still listed as a maintainer? I haven't ran Void Linux in years...

In your log, Gradle is running, so the problem is not with the presence of a Java environment. If you get a reproduction going, `gradle --debug --stacktrace` should provide more information.

adigitoleo commented:
> I'm still listed as a maintainer? I haven't ran Void Linux in years...

Apologies, I just looked at the comment in the gradle template file.

> In your log, Gradle is running, so the problem is not with the presence of a Java environment. If you get a reproduction going, `gradle --debug --stacktrace` should provide more information.

Thanks for the hint, I'll try to reproduce locally (maybe on a different machine) tomorrow.

adigitoleo commented:
Thanks for the review, @paper42. I've addressed the comments in the latest changes. I can repro the build failure now, but only when using xpbs-src and not when manually setting env vars and using make from the source tree. I've reached out to upstream to see if they can help, but will continue to try my best to fix it.
